src/index.html

```diff
@@ -1,14 +1,18 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="page">
+    <main class="watch">
+      <div class="watch_minute-hand"></div>
+      <div class="watch_second-hand"></div>
+      <div class="watch_center"></div>
```

Comment:
```suggestion
      <div class="watch__minute-hand"></div>
      <div class="watch__second-hand"></div>
      <div class="watch__center"></div>
```
There should be two underscores if you follow the BEM methodology